components/readwise/sources/new-documents/new-documents.mjs

```diff
@@ -0,0 +1,64 @@
+import readwise from "../../readwise.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "readwise-new-documents",
+  name: "New Documents",
+  description: "Emit new Document",
```

Comment:
```suggestion
  description: "Emit new document",
```

Reply:
Changed "Document" to "document" to follow Google's style guide on capitalization.